src/components/widget/RawWidget.js

```diff
@@ -115,9 +115,9 @@ export class RawWidget extends Component {
     );
   };
 
-  handleKeyDown = (e, property, value) => {
+  handleKeyDown = (e, property, value, widgetType) => {
     if ((e.key === 'Enter' || e.key === 'Tab') && !e.shiftKey) {
-      if (e.key === 'Enter') {
+      if (e.key === 'Enter' && widgetType.search(/text/i) > -1) {
```

Comment:
isn't this too fragile?
wondering if it would be better to have a helper function, something like isTextWidgetType(widgetType)...

wdyt?

Reply:
Unless I'll get `widgetIsText: true` or something similar, this won't make any difference. Right now I can only tell the difference by using the name. So if I'll test for `text/longText` or this, really comes down to the same thing imho.